common/utils/src/main/resources/error/error-classes.json

```diff
@@ -1968,6 +1968,11 @@
           "expects one of the units without quotes YEAR, QUARTER, MONTH, WEEK, DAY, DAYOFYEAR, HOUR, MINUTE, SECOND, MILLISECOND, MICROSECOND, but got the string literal <invalidValue>."
         ]
       },
+      "LENGTH" : {
```

Comment:
Not sure this name is appropriate, do you have better idea? @MaxGekk

Reply:
Let's leave `LENGTH`.